openapi.yaml

```diff
@@ -0,0 +1,405 @@
+info:
```

Comment:
thoughts on putting this in a /schemas dir? Root dir works too. Just might get cluttered

Reply:
just cuz I want to put the gql schema there too...

Reply:
Addressed in [1773247](https://github.com/Arize-ai/phoenix/pull/3419/commits/177324745d0849c5c25c85e37d5dd622b9345e7b)